packs/prometheus/config.yml

```diff
@@ -0,0 +1,12 @@
+name: Prometheus from newrelic.com/integration
```

Comment:
I think we should move the `prometheus-node-exporter-mac` under the `prometheus` umbrella

Reply:
This could just be done as part of https://github.com/newrelic/newrelic-observability-packs/issues/129

Reply:
ill leave this for 129 then